### PR TITLE
Add Tilted Scroll Component

### PR DESCRIPTION
**Pull request by Tahermaxse**
- Changed the component name from "Skewed Infinite Scroll" to "Tilted Scroll."
- Added Tailwind CSS code.
- Included custom animation keyframes for the "Tilted Scroll" effect .
- Attached a video demonstrating the component in action.
[Uploading Tilted Scroll video.zip…]()

### CODE REVIEW

**Comment by Tahermaxse** [comment]
All set 👍

**Comment by DavidHDev** [comment]
@Tahermaxse , please leave the PR as OPEN if you want me to review it, also, your video didn't seem to upload correctly, you can just add a GIF of the component, it's easier

**Comment by Tahermaxse** [comment]
Thanks for letting me know! I'll leave the PR open for your review. As for
the video, sorry about that! I'll add a GIF of the component instead.

**Comment by Tahermaxse** [comment]
Tilted Scroll Gif
demo gif uploaded

**Comment by Tahermaxse** [comment]
Tilted Scroll Gif
Scroll demo GIF uploaded

File: src/main.jsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
 import './index.scss'
-
```
Comment (DavidHDev) [inline src/main.jsx:4]: This line was used to separate the third party imports from the others, no need to remove it

File: src/content/Components/Tilted Scroll/TiltedScroll.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
+.container {
```
Comment (DavidHDev) [inline src/content/Components/Tilted Scroll/TiltedScroll.css:1]: Even though this is regular CSS, this file should be a `.scss` file like the rest, it's what the project uses. The only reason we use regular CSS in the components is to be able to deliver them with no preprocessor needed

File: src/constants/ExampleConstants.js [hunk 3]
```diff
@@ -613,6 +615,220 @@ const Fade = ({ children, blur = false }) => {
...
+          '0%': {
+            transform:
+              'rotatex(20deg) rotateZ(-20deg) skewX(20deg) translateZ(0) translateY(0)',
+          },
+          '100%': {
+            transform:
+              'rotatex(20deg) rotateZ(-20deg) skewX(20deg) translateZ(0) translateY(-100%)',
+          },
+        },
+      },
```
Comment (DavidHDev) [inline src/constants/ExampleConstants.js:820]: Closing backtick should be placed on this line, so there are no extra empty lines of code in the example

File: src/constants/ExampleConstants.js [hunk 4]
```diff
@@ -613,6 +615,220 @@ const Fade = ({ children, blur = false }) => {
...
+   `
+
+
+},
+
```
Comment (DavidHDev) [inline src/constants/ExampleConstants.js:825]: No need for all these empty lines, please remove them

File: src/constants/ComponentMap.js [hunk 5]
```diff
@@ -45,6 +46,7 @@ export const componentMap = {
   'star-border': StarBorderDemo,
   'elastic-slider': ElasticSliderDemo,
   'grid-motion': GridMotionDemo,
-  'decay-card': DecayCardDemo
+  'decay-card': DecayCardDemo,
+  'tilted-scroll':TiltedScrollDemo
```
Comment (DavidHDev) [inline src/constants/ComponentMap.js:50]: missing space between `:` and `TiltedScrollDemo`, please respect the formatting

File: package-lock.json [hunk 6]
```diff
@@ -15,7 +15,6 @@
         "@react-spring/web": "^9.7.4",
```
Comment (DavidHDev) [inline package-lock.json:15]: Let's not add package-lock changes in here. If it's needed I'll be the one that does it, remove this file from the PR

File: src/content/Components/Tilted Scroll/TiltedScroll.jsx [hunk 7]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (DavidHDev) [inline src/content/Components/Tilted Scroll/TiltedScroll.jsx:1]: no need for this import, it doesn't do anything

File: src/constants/ExampleConstants.js [hunk 8]
```diff
@@ -613,6 +615,220 @@ const Fade = ({ children, blur = false }) => {
 export default Fade;`
   },
 
+// Tilted Scroll
+tiltedScroll:{
+  usage:`import TiltedScroll from './TiltedScroll'
+
+<TiltedScroll />`,
+
+  code:`import React from 'react';
```
Comment (DavidHDev) [inline src/constants/ExampleConstants.js:624]: also remove the React import here, not needed

**Comment by Tahermaxse** [comment]
Thank you so much for taking the time to guide me through these corrections. I've made all the changes as per your suggestions. I really appreciate your detailed feedback and the opportunity to learn from it. This was my first open-source contribution, and your guidance made it a valuable experience. Thanks again!

**Comment by DavidHDev** [comment]
Looks good @Tahermaxse , just don't forget to revert the `package-lock` changes and then we'll merge it, thank you!

**Comment by Tahermaxse** [comment]
I have reverted the changes to `package-lock.json` as requested. The file is now in its original state, and everything looks good on my end. Please let me know if there's anything else needed.

**Comment by DavidHDev** [comment]
@Tahermaxse as you can see here the package-lock file is still part of your PR, you should make sure to push the changes if you managed to revert it, since it's not reflected in your PR:
Screenshot 2024-08-31 at 12 53 20

**Comment by Tahermaxse** [comment]
@DavidHDev I tried to revert the file but ran into some issues. Could you check if the revert worked, or let me know how to fix it?

**Comment by DavidHDev** [comment]
@Tahermaxse now you deleted the whole file from the repo, which isn't good. The way to do it is to copy the existing file from here:
https://github.com/DavidHDev/react-bits/blob/main/package-lock.json

Then add it to your local fork, and then do another commit and push. This should remove any changes related to that file.

**Comment by Tahermaxse** [comment]
@DavidHDev Sorry for mistakenly removing the `package-lock.json`. I've re-added it as you suggested and pushed the changes. Thanks for your help!

**Review by DavidHDev — APPROVED** [review]

**Comment by DavidHDev** [comment]
Okay @Tahermaxse that looks good now, thanks for your time! Will merge it

**Pull request merged**